src/gwt/src/org/rstudio/studio/client/workbench/commands/CmdConstants.java

```diff
@@ -3457,8 +3457,20 @@ public interface CmdConstants extends Constants {
     String toggleRainbowFencedDivsMenuLabel();
     
     // maximizeConsole
-    @DefaultStringValue("Maximize Console") // $NON-NLS-1$
-    String maximizeConsoleMenuLabel();
+    @DefaultStringValue("Toggle Maximize Console") // $NON-NLS-1$
+    String maximizeConsoleLabel();
+    
+    // maximizeSource
+    @DefaultStringValue("Toggle Maximize Source") // $NON-NLS-1$
+    String maximizeSourceLabel();
```

Comment:
Nit: This might read better as "Toggle Maximize Source Pane"?

Reply:
Will do